pyproject.toml

```diff
@@ -32,8 +32,8 @@ dependencies = [
     "pyyaml>=6.0.1",
     "omegaconf>=2.3.0",
     "submitit>=1.4.6",
-    "typing-extensions==4.5.0",
-    "typing_extensions==4.5.0",
+    "typing-extensions>=4.5.0",
```

Comment:
These '==' were there because of some bug introduced in later versions. But we can try to remove/comment these lines, as the bug may have been solved by now

Reply:
same reason: needed for the NGC container